app/frontend/stylesheets/object/component/_modal-window.sass

```
@@ -0,0 +1,33 @@
+.modal-window
+  position: fixed
+  top: 0
+  left: 0
+  width: 100%
+  height: 100%
+  background-color: $COLOR_SEPARATOR
+  z-index :100000
+  .message-area
+    position: absolute
+    top: 50%
+    left: 50%
+    transform: translate(-50%,-50%)
+    padding: 0.8rem 1.5rem 0.8rem 1rem
+    background-color: #fff
+    border-radius: 3px
+    border: 1px solid $COLOR_DARKER_GRAY
+    font-size: 0.8rem
+    &:before
+      content: $CHAR_EXCRAMATION_TRIANGLE
+      font-family: FontAwesome
+      color: var(--color-sign-warning)
```

すごく細かいですが、他の場所には全てSASS の variable 使っていますが、ここだけは、なぜか CSS の variable になっていますので、ここにも、SASS のを使ってもいいと思います：
```suggestion
      color: $COLOR_SIGN_WARNING
```